includes/Modules/Search_Console.php

```diff
@@ -553,4 +558,14 @@ protected function setup_assets() {
 		);
 	}
 
+	/**
+	 * Returns TRUE to indicate that this module should be always active.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool Returns `true` indicating that this module should be activated all the time.
+	 */
+	public static function is_force_active() {
+		return true;
+	}
```

Comment:
Now that this method is used to mark a module as `force_active`, we should probably remove the duplicate directive within the `Search_Console::setup_info` method. It will already be force-active because of this new method here, and otherwise we'll have to maintain this in two places.